assets/src/edit-story/components/carousel/pagepreview/utils.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import {
+  PAGE_WIDTH,
+  FULLBLEED_HEIGHT,
+  DANGER_ZONE_HEIGHT,
+} from '../../../constants';
+
+export function getBox({
```

Comment:
Maybe all the SVG-related things would deserve their own folder instead of being part of `pagepreview`? Doesn't feel directly related to the page thumbnails, assuming it will be used for canvas, too.

Reply:
Very true, that's also a part of the future work to be done on this.

Reply:
Issue to follow up on this: https://github.com/google/web-stories-wp/issues/7636